### ISSUE TITLE
Dear author，how can I get gif only?

### ISSUE THREAD

**Issue by brandy2015** [tilltue/TLPhotoPicker#145]

**Comment by tilltue** [comment]
@brandy2015 

My guess is there is not options of filename ( or filename extension ) fetchOptions.

https://developer.apple.com/documentation/photokit/phfetchoptions#1965657

<img width="775" alt="2018-10-23 11 24 46" src="https://user-images.githubusercontent.com/2108446/47331244-497b2800-d6b6-11e8-864c-bd375f6c057a.png">

PHAsset 'Supported Keys' : unsupported predicated descriptior key  "filename"

**Comment by brandy2015** [comment]
Is there any easy way to display gif only?

**Comment by brandy2015** [comment]
Or  How can I put  PHFetchResult to TLPhotoPicker to display?

func getGif() -> PHFetchResult<PHAsset> {
        if let gifCollection = PHAssetCollection.fetchAssetCollections(with: .smartAlbum, subtype: .smartAlbumAnimated, options: nil).firstObject {
            return PHAsset.fetchAssets(in: gifCollection, options: nil)
        }
        return PHFetchResult<PHAsset>()
    }

**Comment by tilltue** [comment]
Oh! I See. 👍 
I will be fetchAssetCollections configure options.
maybe like this

```swift 3
public struct TLPhotosPickerConfigure {
        public var fetchCollectionTypes: [(PHAssetCollectionType,PHAssetCollectionSubtype)]? = nil
}
//used
configure.fetchCollectionSubTypes = [(.smartAlbum, .smartAlbumAnimated)]
//expected only put the .smartAlbumAnimated
```

**Comment by brandy2015** [comment]
Yes!

**Comment by brandy2015** [comment]
Maybe set some options to select gif type directly and conveniently

**Comment by brandy2015** [comment]
Thanks，I will see it

**Comment by brandy2015** [comment]
It's really awesome ! That's exactly what I want. Thank you very much！ But how can I set it to default display? The picture below



![img_4657](https://user-images.githubusercontent.com/10484324/47567040-0f34b380-d960-11e8-92ba-f96a00e233c7.PNG)
![img_4658](https://user-images.githubusercontent.com/10484324/47567042-1065e080-d960-11e8-9bc9-186844c3984a.PNG)

**Comment by tilltue** [comment]
@brandy2015 
I've fixed that 4145c19568c1922948fea7b0cef6b6a148845c04

**Comment by tilltue** [comment]
i've updated 1.7.7 😄
